net.sf.eclipse.tomcat/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,8 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.debug.ui,
  org.eclipse.ui.ide,
  org.eclipse.core.variables,
- com.google.guava;bundle-version="15.0.0"
+ com.google.guava;bundle-version="15.0.0",
+ org.eclipse.wst.xml.core
```

Comment:
Please make this depdendency optional by adding _resolution:=optional_ in manifest or via corresponding IDE setting.